src/logger.js

```diff
@@ -1,39 +1,43 @@
 // logger.js
 
-var path = require('path');
 var _ = require('underscore');
-var winston = require('winston');
-var stackTrace = require('stack-trace');
+
+var levels = [
+    'debug',
+    'dir',
+    'log',
+    'info',
+    'warn',
+    'error',
+    'assert'
+];
 
 /**
  * Creates a custom logger for this package
  */
-module.exports = new winston.Logger({
-    transports: [
-        new winston.transports.Console({
-            name: 'console',
-            level: 'warn',
-            colorize: true
-        }),
-        new winston.transports.File({
-            name: 'rdf-components',
-            filename: 'rdf-components.log',
-            level: 'info'
-        })
-    ]
-});
+module.exports = {
+    time: console.time.bind(console),
+    timeEnd: console.timeEnd.bind(console),
+    silence: function(limit) {
+        levels.forEach(function(level) {
+            if (!limit || limit == 'all' || levels.indexOf(level) <= levels.indexOf(limit))
+                this[level] = _.noop;
```

Comment:
Would it be helpful (i.e., easier for callers) to save the previous logging level (by pushing it onto a stack) when 'silence' is called, so that it can restored later (by popping it) without having to know what it was before?  Usage something like this:

```
silence();   // Temporarily quiet the logging
. . . Do noisy stuff here . . . 
unsilence();   // Restore the previous logging level
```